reports/Arkansas.md

```diff
@@ -1,19 +1,28 @@
 ## Bentonville
 
-### Law enforcement gas a crowd chanting “we want peace” right after exiting the building. | June 1st
+### Law enforcement tear-gas a crowd chanting “we want peace” right after exiting the building. | June 1st
 
-Law enforcement gas a crowd chanting “we want peace” right after exiting the building, in the video there was no order to disperse, the crowd seemed to be peacefully chanting.
-Article says crowd started throwing water bottles and firecrackers. Sheriff's office claims rocks were thrown.
+A crowd gathered on the Bentonville Square to protest. Around 9 PM, law enforcement gassed a crowd chanting “we want peace.” Local news says some people throwing water bottles and firecrackers. Police also claim rocks were thrown. Police claim that at least one police vehicle was damaged. Local news reports vary -- some reports have protesters "kicking" a police vehicle; some televised reports have protesters "damaging" a police vehicle; most written articles merely report that police say protesters damaged a police vehicle. Assuming that a police vehicle was in fact damaged, it is unclear to what extent (though per one op-ed writer, the police chief says $8000 worth of damage was done).
 
-tags: tear-gas, tear-gas-canister
+It is unclear whether any order to disperse was given before the first use of tear gas; in the videos purporting to be from that time, no such order is audible. (In one video -- see third link -- a voice can be heard that sounds like it comes from a PA system, but the words are unclear.) Several people present, including one op-ed writer, claim not to have heard any such order. Orders to disperse were certainly given repeatedly between 9:00 and 11:00, as police continued to deploy tear gas.
+
+Police posted on the City of Bentonville Facebook page that rubber bullets were not used. That post was incorrect.
```

Comment:
Can you direct to the link that verifies rubber bullets were used?

Reply:
The 5 News link. The op ed says that the Bentonville police chief said the error was due to a communication failure. It is, of course, possible that the only rubber bullets used were used after the Facebook post. I think that's unlikely, but I'd be open to editing the description to note the possibility.

Reply:
(Just checked the time stamp -- the denial was posted at 10:53 local time. Since reports claim most of the protesters were gone by 11, it seems likely the denial was posted after the rubber bullets were used.)

Reply:
Okay maybe mention that the news article states someone was shot with rubber bullets?